### PR TITLE
Update wkcuber for typing support

### PR DESCRIPTION
**Pull request by rschwanhold**

### CODE REVIEW

File: wkconnect/utils/types.py [hunk 1]
```diff
@@ -155,5 +155,5 @@ def np_slice(self) -> np.lib.index_tricks.IndexExpression:
 
 
 class HashableDict(dict):
-    def __hash__(self) -> int:
+    def __hash__(self) -> int:  # type: ignore  # signature incompatible with supertype
```
[the remark this answers is not in the document]
Comment (jstriebel) [reply wkconnect/utils/types.py:158]: No, not really, this is weird. Maybe making it a `HashableDict(dict, typing.Hashable)` helps? But just wild guessing. `int` is definitely correct for objects, but maybe `dict` sets the return type to `None`, which would not allow int for children anymore. You could also try to use only `typing.MutableMapping` or `typing.Dict` as a parent.

[the remark this answers is not in the document]
Comment (fm3) [reply wkconnect/utils/types.py]: Unfortunately, pylint does not let us inherit from anything from Typing (` Inheriting 'Hashable', which is not a class. (inherit-non-class)`).

Why is there a need of a hashable dict anyway? Seems somewhat dangerous as it is still mutable. Maybe something else can be used?

[the remark this answers is not in the document]
Comment (jstriebel) [reply wkconnect/utils/types.py]: Good point. Seems like it is only used for the boss channels (in experiment model). The only usage (except from hashing it) is [here](https://github.com/scalableminds/webknossos-connect/blob/master/wkconnect/backends/boss/backend.py#L131). We could use an immutable dict implementation like the top answer here instead: https://stackoverflow.com/questions/2703599/what-would-a-frozen-dict-be

[the remark this answers is not in the document]
Comment (fm3) [reply wkconnect/utils/types.py]: I had a look but more new problems arose from trying to properly type that FrozenDict, so I’d say let’s go with the HashableDict for now, as discussed

**Review by fm3 — APPROVED** [review]

**Pull request merged**